internal/driver/interactive.go

```diff
@@ -82,9 +88,13 @@ func interactive(p *profile.Profile, o *plugin.Options) error {
 					continue
 				}
 				// Allow group=variable syntax by converting into variable="".
-				if v := pprofVariables[value]; v != nil && v.group == name {
-					if err := pprofVariables.set(value, ""); err != nil {
-						o.UI.PrintErr(err)
+				if groups[name] {
+					if v := pprofVariables[value]; v != nil && v.group == name {
+						if err := pprofVariables.set(value, ""); err != nil {
+							o.UI.PrintErr(err)
+						}
+					} else {
+						o.UI.PrintErr(fmt.Errorf("Unrecognized value for %s: %q", name, value))
```

Comment:
It might be good to print out the list of values that are recognized?

Reply:
Added that.